tests/testthat/test-object_usage_linter.R

```diff
@@ -480,3 +480,15 @@ test_that("unknown infix operators give good lint metadata", {
     object_usage_linter()
   )
 })
+
+test_that("respects `skip_with` argument for `with()` expressions", {
+  f <- tempfile()
```

Comment:
this `f` is not cleaned up afterwards.

better to use `withr::local_tempfile(lines = c("test_fun...", ...))`

Reply:
Good point. Changed to use `{withr}`.